source/gui/settingsDialogs.py

```diff
@@ -2171,10 +2194,15 @@ def onSave(self):
 		log.debug("Saving advanced config")
 		config.conf["development"]["enableScratchpadDir"]=self.scratchpadCheckBox.IsChecked()
 		config.conf["UIA"]["useInMSWordWhenAvailable"]=self.UIAInMSWordCheckBox.IsChecked()
-		if self.ConsoleUIACheckBox.IsChecked():
-			config.conf['UIA']['winConsoleImplementation'] = "UIA"
-		else:
-			config.conf['UIA']['winConsoleImplementation'] = "auto"
+		consoleChoice = self.consoleCombo.GetSelection()
+		consoleIndicesToVals = {

```

Comment:
Please remove this in favour of the list as suggested above. You can save that list as an attribute on the panel. There are numerous examples within the NVDA code base how this is handled, e.g. focus context presentation.